packages/genesys-spark-components/src/components/beta/gux-rich-text-editor/gux-rich-text-editor-beta.tsx

```diff
@@ -288,12 +298,25 @@ export class GuxRichTextEditor {
     return this.renderSlot('global-action', 'gux-global-action-container');
   }
 
+  private hasToolbarChildren(): boolean {
+    const children = [
+      this.renderTypographicalEmphasis(),
+      this.renderTextStyling(),
+      this.renderListsIndentation(),
+      this.renderInserting(),
+      this.renderTextEditorMenu(),
+      this.renderGlobalAction()
+    ].filter(child => child !== null && child !== undefined);
+    return children.length > 0;
+  }
+
   render(): JSX.Element {
     return (
       <div
         class={{
           'gux-rich-text-editor-container': true,
-          'gux-disabled': this.disabled
+          'gux-disabled': this.disabled,
+          'gux-toolbar-hidden': !this.hasToolbar
```

Comment:
I could have just conditionally rendered the `gux-rich-text-editor-toolbar-container` aswell but this will affect all the resizing logic so instead of adding additional `if` guards around the code I think this is the least invasive way to just remove the gap. I have also tested dynamically addding elements to the toolbar and it works as expected.